lib/api/eval-response.js

```diff
@@ -0,0 +1,30 @@
+import URI from 'urijs';
+
+// make these available for eval
+import _faker from 'faker/build/build/faker';
+import _state from 'api/local-state';
+
+export function evalResponse(responseRule, request) {
+    // Setup local variables for eval
+  let url = URI(request.url);
+  let path = url.segmentCoded();
+  let query = url.search(true);
+  let data = JSON.parse(request.body);
```

Comment:
It's not form-data, it's just a request with data that's not in JSON format, which is legit (as Maayan says).